src/components/form/LocationForm.js

```diff
@@ -204,9 +205,13 @@ export const LocationForm = ({ desktop }) => {
       unverified: false,
     }
 
-    console.log('locationValues', locationValues)
-    const locationResp = await addLocation(locationValues)
-    console.log('locationResp', locationResp)
+    let locationResp
+    try {
+      locationResp = await addLocation(locationValues)
+      toast.success('Location submitted successfully!')
+    } catch {
+      toast.error('Location submission failed.')
```

Comment:
Maybe we could return a redirect to the /map page or something here on error since we could cause an error on line 223 (copied below) where we access an attribute of `locationResp`

```js
const reviewResp = await addReview(locationResp.id, reviewValues)
```

Reply:
> Maybe we could return a redirect to the /map page or something here on error since we could cause an error on line 223 (copied below) where we access an attribute of locationResp

Wouldn't it be a weird experience if you submit a form and it redirects on failure? Like you'd probably want it to stay on the same @SirajChokshi